### PR TITLE
Get name from API

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/pages/common/dashboard/controller.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const getMarketingPageDashboardContext = async ({
     const dashboardData = dashboardDataRaw.data;
     const context = createDashboardPageContext({
       solutionId,
+      solutionName: dashboardData.name,
```
Comment (kat1906) [inline app/pages/common/dashboard/controller.js]: please add test to controller.test.js

File: app/pages/common/dashboard/createDashboardPageContext.js [hunk 2]
```diff
@@ -125,6 +125,7 @@ export const createDashboardPageContext = ({
   const isSupplierContextType = userContextType === 'supplier';
 
   const context = {
+    name: solutionName,
```
Comment (kat1906) [inline app/pages/common/dashboard/createDashboardPageContext.js]: please add test to createDashboardPageContext.test.js

File: app/pages/supplier/dashboard/template.njk [hunk 3]
```diff
@@ -23,7 +23,7 @@
       }) }}
 
       {% if title %}
-        <h1 data-test-id="dashboard-title">{{ title }}</h1>
+        <h1 data-test-id="dashboard-title">{{ name }} {{ title }}</h1>
```
Comment (kat1906) [inline app/pages/supplier/dashboard/template.njk]: Please add test to the template.test.js

Comment (BenBJSS) [reply app/pages/supplier/dashboard/template.njk]: There is already a test for this

Comment (kat1906) [reply app/pages/supplier/dashboard/template.njk]: ah yeah because they are together 👍

**Pull request merged**